doc/md/cloud/setup-ci.mdx

```diff
@@ -0,0 +1,322 @@
+---
+id: setup-ci
+title: Setting up CI for your Database with Atlas Cloud
+---
+
+As your application evolves, so does your database schema. If you're not careful
+with schema changes (migrations) you can end up introducing all sorts of issues
+that are painful and expensive to fix.
+
+To mitigate the risk of deploying dangerous changes to database schemas, many teams
+apply CI/CD practices to their database. This means that every change to the database
+schema is automatically reviewed and tested before it is deployed to production.
+
+In this guide, we'll show you how to set up CI for your database using Atlas Cloud.
+
+## Push or Pull Workflows
+
+There are two ways to set up CI for your database with Atlas Cloud:
+
+* **Pull (GitHub App)** - You install the Atlas Cloud GitHub App on your GitHub
+  repository and Atlas Cloud will automatically pull changes from your GitHub
```

Comment:
```suggestion
  repository. Atlas Cloud will automatically pull changes from your GitHub
```

Reply:
Breaking up a long sentence makes it easier to read. Otherwise, the sentence is fine.